sphinxcontrib/matlab.py

```diff
@@ -823,3 +823,6 @@ def setup(app):
 
     app.add_autodoc_attrgetter(doc.MatModule, doc.MatModule.getter)
     app.add_autodoc_attrgetter(doc.MatClass, doc.MatClass.getter)
+
+    return {'paralell_read_safe':False}
```

Comment:
Typo: probably should be `parallel_read_safe`

Reply:
You're right about the typo, thanks!